accessibility-checker-engine/help/IBMA_Color_Contrast_WCAG2AA_PV.mdx

```diff
@@ -12,19 +12,19 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 
 * People with moderately low vision who do not use a contrast-enhancing technology 
 * People who do not see the full range of colors 
-* People who see no color. 
+* People who see no color
 
 ## Why is this important?
 
-When the foreground text and its background colors have less than a 4.5:1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology.
+When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. For larger text of 18 point or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.
 
 </Column>
 <Column colLg={11} colMd={5} colSm={4} className="toolMain">
 
 <div id="locLevel"></div>
 
-## Verify that image layers or color gradients do not reduce the text contrast (calculated at {0}) below WCAG 2.1 AA requirements for text of size {1}px and weight of {2}
-The contrast ratio of text and the background (i.e. background with a color gradient or a background image) must meet WCAG 2.1 AA requirements 
+## Verify the contrast ratio of the text against the lightest and the darkest colors of the background meets the WCAG 2.1 AA minimum requirements for text of size {1}px and weight of {2} 
+The contrast ratio of text with a color gradient or a background image must meet WCAG 2.1 AA requirements 
```

Comment:
I think it would be clearer to say 'text against a color gradient' or else it could be interpreted as the text having the color gradient.